webapp/components/Comment/Comment.vue

```diff
@@ -89,9 +69,14 @@ import scrollToAnchor from '~/mixins/scrollToAnchor.js'
 
 export default {
   mixins: [scrollToAnchor],
-  data: function() {
+  data() {
+    const anchor = `commentId-${this.comment.id}`
+    const isTarget = this.$route.hash === `#${anchor}`
```

Comment:
I would prefer not to have `this.$route` in components. Is it possible to pass this as a prop?